http/src/main/scala/filodb/http/FiloHttpServer.scala

```diff
@@ -39,9 +42,14 @@ class FiloHttpServer(actorSystem: ActorSystem) extends StrictLogging {
     val bindingFuture = Http().bindAndHandle(finalRoute,
       settings.httpServerBindHost,
       settings.httpServerBindPort)
-    val bind = Await.result(bindingFuture,
+    binding = Await.result(bindingFuture,
       scala.concurrent.duration.Duration.fromNanos(settings.httpServerStartTimeout.toNanos))
-    logger.info("FiloDB HTTP server is live at http:/{}/", bind.localAddress)
+    logger.info("FiloDB HTTP server is live at http:/{}/", binding.localAddress)
+  }
+
+  def shutdown(hardDeadline: FiniteDuration): Future[Http.HttpTerminated] = {
```

Comment:
Is this an unrelated change? Why is it needed?

Reply:
This is part of some trial and errors to get IngestionAndRecoverySpec multi-jvm spec to run.
I am finding that the ports are not being released in the restart step of the tests.

Reply:
Rather than shutting down previous ports we should just try to allocate a different, unique port for each test.